src/App.js

```diff
@@ -1,27 +1,33 @@
-import logo from './logo.svg';
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/158276403-9de335e0-ee6f-4c15-a488-b51ce07d41c4.png)

Therefore:

- [ ] All over the project there is code left behind from the CRA when the app was created. Kindly please remove all the texts, images, styles, and tests left by the CRA.

- Most of it is in the public folder.
- Also, `App.test.js` should be deleted.
- setupTests.js
- reportWebVitals.js
- The react logo images on /public
- If there is CSS code you didn't write, please remove it

That way, the repo will be a lot cleaner and we will meet the requirement💪